DS13/code/modules/client/preferences.dm

```diff
@@ -0,0 +1,189 @@
+/datum/preferences
+	var/backstory_reason = "I joined starfleet to find purpose.<br>"
+	var/backstory_academy = "At the Academy, I was a very average student. I paid attention in class and turned up when required but that was the limit.<br>"
+	var/backstory_experience = "I'm a seasoned officer, I've been on multiple starship tours and am pretty competant.<br>"
+	var/backstory_upbringing = "I had an easy childhood and always got what I needed and sometimes, what I wanted.<br>"
+	var/backstory_parents = "My parents worked at Starfleet headquarters in secretarial positions, their careers weren't really important.<br>"
+	var/backstory_accepted = "I loved my parents and have many fond memories of them.<br>"
+	var/backstory_ambition = "Nothing"
+
+/datum/mind
+	var/backstory_set = FALSE //Don't CLOG UP MEMORIES with INFINITE COPIES OF BACKSTORY AAAAHHHHH
+
+/datum/asset/simple/jobpreviews
+	assets = list(
+		"command.png"	= 'DS13/icons/previews/command.png',
+		"engsec.png"	= 'DS13/icons/previews/engsec.png',
+		"medsci.png"	= 'DS13/icons/previews/medsci.png')
+
+/datum/preferences/proc/create_backstory(var/mob/user)
+	var/responses = list("keen", "entitled", "neutral", "lazy", "rebel", "insane")
+	var/choice = input(user,"Why did you join starfleet?", "Backstory", null) in responses
+	if(!choice)
+		return
+	switch(choice)
+		if("keen")
+			backstory_reason = "I joined starfleet to uphold the principles of the federation!<br>"
+		if("entitled")
+			backstory_reason = "I joined starfleet because I deserve a good job.<br>"
+		if("neutral")
+			backstory_reason = "I joined starfleet to find purpose.<br>"
+		if("lazy")
+			backstory_reason = "I joined starfleet because my parents told me to.<br>"
+		if("rebel")
+			backstory_reason = "I joined starfleet to find somewhere I belong.<br>"
+		if("insane")
+			backstory_reason = "I joined starfleet to kill hostile aliens.<br>"
+	choice = input(user,"What were you like at starfleet academy?", "Backstory", null) in responses
+	if(!choice)
+		return
+	switch(choice)
+		if("keen")
+			backstory_academy = "At the Academy, I worked hard and received top marks in everything I did. It wasn't easy, but it was worth it. I was put on the fast track to the starfleet command training program.<br>"
```

Comment:
change it so backstory_academy already has "At the Academy, " and do a backstory_academy += "I worked hard etc etc" Saves slightly repitition.

Reply:
initial(academy) + thing